src/index.scss

```diff
@@ -8,3 +8,7 @@ The React <Content /> component uses the `main` element which IE11 doesn't recog
 main {
   display: block;
 }
+
+$feature-flags: (
+  ui-shell: true,
+);
```

Comment:
The feature flags mapping should come before the styles import. See https://www.carbondesignsystem.com/tutorial/react/step-1#add-ui-shell-sass